buy-resell/products/digital-receipt.md

```diff
@@ -0,0 +1,57 @@
+---
+slug: /posdealers/buy-resell/products/digital-receipt
+title: Digital Receipt
+---
+
+# Digital Receipt
+
+:::info summary
+
+After reading this, you can explain the preparations for PosOperators to use digital receipts.
+
+:::
+
+## Introduction
+
+If you include digital receipts in the sales process, the customer can choose between a digital receipt and a printout.  
+With digital receipts, the PosOperator can reduce expenses for printouts and help the environment.  
+As a PosDealer, you thus offer a state-of-the-art PosSystem.  
+
+
+![receipt-chaining](../images/buy_resell-digital-receipt.png)
+
+Please note that the visualization in the sandbox may look different than in the productive system.
+Please also note that different regulations or requirements may apply depending on the country where your PosSystem is used. Please refer to the country-specific regulations. Please understand that we can only describe these details without claiming to be complete.
+
+## Preparations general
+
+PosDealers or PosOperators can make the preparations for having digital receipts available. For this, only access to the fiskaltrust.Portal with **Write authorizations** is required.
+The Master Data of the PosOperators account in the fiskaltrust.Portal must be complete. The stored tax identification numbers should be verified.
+The details of the Outlet, not the company, are used as the address on the digital receipt. Therefore, these should be checked for completeness and corrected if necessary. 
+The digital receipt may list a logo at the top. If this logo is desired, you should add it to the data of the Outlet. 
+We recommend using a file in ~.png-format with transparent background for best results. 
+
+## Preparations country-specific
```

Comment:
I assume it is already clarified that these operations will vary from country to country? 
If the main steps would be the same, it would be cool if we could describe it without country specific parts. If they are necessary - will the digital receipt team create the content for them ?

Reply:
At this moment we refer to the existing description, that was made for onboarding. If the digital receipt team need more details, they can add them there.